packages/theme/src/abstracts/_variables-bootstrap.scss

```diff
@@ -11,25 +11,25 @@ $white: #fff;
 $black: #000;
 
 // Gray
-$gray-base: #576373;
+$gray-base: #5c5c5c;
 $gray-darker: #344154;
 $gray-dark: #758397;
 $gray: #9dafbd;
-$gray-light: #9dafbd;
-$gray-lighter: #d7dbe0; // lighten($gray-base, 93.5%) !default; // DDE0E6
+$gray-light: #999999;
```

Comment:
Why are the color variables here different than the _colors.scss?

Reply:
Fixing _colors.scss to be consistent.